### PR TITLE
cplp-1310 - get all partner companies bpn

### PR DESCRIPTION
**Pull request by Akshay-Chaudhari-BMW**
get all partner companies bpn

### CODE REVIEW

File: src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/PartnerNetworkController.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    /// <summary>
+    /// Get all member companies
+    /// </summary>
+    /// <returns>Returns all the active member companies bpn.</returns>
+    /// <remarks>
+    /// Example: Get: api/administration/partnernetwork/memberCompanies
+    /// </remarks>
+    /// <response code="200">Returns all the active member companies bpn.</response>
+
```
Comment (Phil91) [inline src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/PartnerNetworkController.cs:37]: Please remove empty line

**Review by Phil91 — DISMISSED** [review]
Overall it looks good. Could you provide some unit tests for it, that we match our code coverage goal?

File: src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/PartnerNetworkController.cs (hunk 1 above)
Comment (Akshay-Chaudhari-BMW) [reply src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/PartnerNetworkController.cs:33]: resolved

Comment (jjeroch) [reply src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/PartnerNetworkController.cs:33]: successfully rechecked

**Review by jjeroch — APPROVED** [review]
@Phil91 I approved it already to be able to proceed with the FE part, but please feel free to a quick look next week at the implementation if you see any necessary change needs

**Pull request merged**